server/src/main/java/org/opensearch/cluster/coordination/FollowersChecker.java

```diff
@@ -159,6 +162,10 @@ public void onNodeDisconnected(DiscoveryNode node, Transport.Connection connecti
         });
     }
 
+    private void setFollowerCheckTimeout(TimeValue newValue) {
+        this.followerCheckTimeout = newValue;
+    }
```

Comment:
Should we add some checks to validate the `newValue`? For example, enforcing a minimum/maximum timeout value that we want to accept

Reply:
Thank you for your input. I will add validation tests into the project. Additionally, although we already have a minimum timeout value check, I propose setting a conservative maximum timeout value of 30 seconds. However, considering the potential for wider use cases, increasing it to 60 seconds seems more accommodating. What is your take on this?